paddle/fluid/operators/reduce_ops/reduce_mean_op.cc

```diff
@@ -13,8 +13,74 @@
 // limitations under the License.
 
 #include "paddle/fluid/operators/reduce_ops/reduce_mean_op.h"
+#include <memory>
+#include <string>
+#include <vector>
 
-REGISTER_REDUCE_OP(reduce_mean);
+namespace paddle {
+namespace operators {
+
+class ReduceMeanOpGradDescMaker : public framework::SingleGradOpDescMaker {
```

Comment:
Add comments for why rewrite the `ReduceMeanOpGradDescMaker`

Reply:
Done